src/libPMacc/include/compileTime/AllCombinations.hpp

```diff
@@ -90,6 +92,7 @@ struct AllCombinations<T_MplSeq, T_TmpResult, false >
 
     BOOST_STATIC_CONSTEXPR uint32_t rangeVectorSize = bmpl::size<MplSeq>::value;
     typedef typename bmpl::at<MplSeq, bmpl::integral_c<uint32_t, rangeVectorSize - 1 > > ::type LastElement;
+    typedef bmpl::empty<LastElement> IsLastElementEmpty;
```

Comment:
The reason I merged this, was that I didn't get any response, so I thought: ok seems fine, no big deal.